.circleci/ci-packages.txt

```diff
@@ -8,7 +8,6 @@ libaspell15-0.60.8
 libcmark0_28_3-0.28.3
 libjq1-1.6
 libonig4-6.7.0
-libopencv3_4-3.4.9
```

Comment:
I don't understand why this is being removed - @Martchus since you were looking into the packaging, can you confirm this is expected?